src/pages/search/project_search/ProjectListItem.tsx

```diff
@@ -28,6 +29,11 @@ export const ProjectListItem = ({ project, refetchProjects, showEdit = false }:
 
   return (
     <SearchListItem sx={{ borderLeftColor: 'project.main' }}>
+      <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', mb: '0.5rem' }}>
```

Comment:
Dette var en liten endring som utgjorde stor forskjell! 🎉 
Litt redd folk kommer til å se på det som en bug at ikonet mangler for resultat og person da, siden det egentlig skal være på alle 3

Reply:
Ja, enig, vi burde få dem på alle snarest!